pyramid_swagger/tween.py

```diff
@@ -93,6 +99,34 @@ def _get_validation_context(registry):
         return noop_context
 
 
+def get_swagger_handler(settings, route_info, registry):
+    """Contains callables that isolate implementation differences in the tween
+    to handle both Swagger 1.2 and Swagger 2.0.
+
+    Exception is made when `settings.include_2dot0_routes` are non-empty and
```

Comment:
I think it is helpful to keep it in the method docstring. (comments inside the method wont reflect)